router/forwarder/forwarder.go

```diff
@@ -102,8 +102,7 @@ func (forwarder *Forwarder) StartDestinations(sessionId *identity.TokenId) {
 		for _, address := range addresses {
 			if destination, found := forwarder.destinations.getDestination(address); found {
 				if xgressDest, ok := destination.(XgressDestination); ok && xgressDest.IsTerminator() {
-					xgressDest.Start()
-					pfxlog.Logger().Debugf("started Xgress for session %v", sessionId.Token)
+					pfxlog.Logger().Debugf("[deprecated call to started Xgress for session %v]", sessionId.Token)
```

Comment:
Just for education purposes while debugging?

Reply:
I was going to leave some things with an eye towards backwards compatibility, but then I changed my mind. I'll go through and rip the rest of this out as well.

Reply:
done